src/completers/docker.zsh

```diff
@@ -32,3 +52,19 @@ _fzf_complete_docker-images() {
 _fzf_complete_docker-images_post() {
     awk '{ print $1 }'
 }
+
+_fzf_complete_docker-containers() {
+    local docker_options=$1
+    local fzf_options=$2
+    shift 2
+
+    _fzf_complete "--ansi --tiebreak=index --header-lines=1 $fzf_options" $@ < <(
+        docker container list ${(Z+n+)docker_options} \
+            --format 'table {{.ID}};{{.Image}};{{.Command}};{{.RunningFor}};{{.Status}};{{.Ports}};{{.Names}}' 2> /dev/null \
+                | FS=';' _fzf_complete_tabularize $fg[yellow] $reset_color{,,,,}
+    )
+}
+
+_fzf_complete_docker-containers_post() {
+    awk '{ print $1 }'
```

Comment:
Optional: `docker cp` actually takes `CONTAINER:SRC_PATH` as you cited. The whitespace after `docker cp **<TAB>` is executed is quite useless because a user anyway removes that and type `:`. Probably we can just drop support for `docker cp **<TAB>` for now or implement this post function more conveniently. Which way do we go?

Reply:
Let's drop support for it for now.

Reply:
Just remove all the code around `docker cp` then.